ztp/gitops-subscriptions/argocd/example/policygentemplates/group-du-sno-ranGen.yaml

```diff
@@ -94,6 +94,31 @@ spec:
             fsType: xfs
             devicePaths:
               - /dev/sdb2
+    # manually create PVs if they are not dynamically created by local-storage-operator
+    - fileName: StoragePV.yaml
+      policyName: "config-policy"
+      metadata:
+        name: pv-storage-ptp
+      spec:
+        storageClassName: "example-storage-class-1"
+        capacity:
+          storage: 10Mi
+        local:
+          # manually create ptp directory if local-storage is shared with bmer
+          # path: /mnt/local-storage/ptp
+          path: /mnt/local-storage
+    - fileName: StoragePV.yaml
+      policyName: "config-policy"
+      metadata:
+        name: pv-storage-bmer
```

Comment:
in current ztp examples, bmer related configs are missing. So we need to do one of the following:

1. add all the bmer related configs in ztp example (operator deployment, create pv, create hardwareevent CR); or 
2. do not add any bmer related configs.

@imiller0 any suggestions?

Reply:
updated, added bmer with default amqp transport. HTTP transport policies are commented out.